its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -991,6 +991,39 @@ void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
       .doesNotContain("WithChanges.cs"); // Was modified
   }
 
+  @Test
+  void checkMultiLanguageSupportWithSdkFormat() throws Exception {
+    if (!TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022")) {
+      return; // This test is not supported on versions older than Visual Studio 22
```

Comment:
Why do we need this restriction?

Reply:
After our discussion, I took a closer look.
The new SDK-style format for .csproj files was introduced with .NET Core. The project is targeting .net7 so both MSBuild15 and MSBuild16 QA were failing:
- MSBuild16 --> VS2019 I should target .net5 and I assume it will work
- MSBuild15 --> VS2017 we are missing the .NET Core SDK, so it's a bit tricky

I might just target .net5 and reduce the min version requirements for the test to VS2019. Does it work for you?